### PR TITLE
Add VectorGraphics.CanvasView Test

### PR DESCRIPTION
**Pull request by JSUYA**
Basic Shape is the first.
Add more tests(Gradient, clipping , etc..) later.

Screenshot from 2021-09-10 17-17-23

### CODE REVIEW

File: NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1Page.xaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<ContentPage x:Class="NUITizenGallery.VectorGraphicsCanvasViewTest1Page"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+  xmlns:base="clr-namespace:Tizen.NUI.BaseComponents.VectorGraphics;assembly=Tizen.NUI"
```
Comment (everLEEst) [inline NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1Page.xaml:5]: vector would be better than base... but not important thing..

**Review by everLEEst — APPROVED** [review]
LGTM.
only namespace in xaml looks wierd.. but its okay I think.

File: NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+using Tizen.NUI.Components;
+
+namespace NUITizenGallery
+{
+    internal class VectorGraphicsCanvasViewTest1 : IExample
+    {
+        private Window window;
+        public void Activate()
+        {
+            Console.WriteLine($"@@@ this.GetType().Name={this.GetType().Name}, Activate()");
```
Comment (sparrow74) [inline NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1.cs:29]: Why this comment as "@@@" is added to log?

**Review by huiyueun — APPROVED** [review]

File: NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1.cs (hunk 2 above)
Comment (everLEEst) [reply NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1.cs:29]: This print is all tests commonly have in window activation... not sure who put @@@ at the first time...

**Comment by JSUYA** [comment]
ok please someone push merge button

**Comment by JSUYA** [comment]
or give me permission :)

File: NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1Page.xaml (hunk 1 above)
Comment (JSUYA) [reply NUITizenGallery/Examples/VectorGraphicsTest/VectorGraphicsCanvasViewTest1Page.xaml:5]: It's just taken from other code.
I'll fix it when I add the next test.

**Pull request merged**